.github/workflows/label-new-issues.yml

```
@@ -0,0 +1,36 @@
+# Description: This workflow automatically adds a `pending-triage` label to issues that need attention
+#
+# Triggered by: newly opened, re-opened or transferred issues
+
+name: Mark issues for triage
```

Nittish: Existing workflows _mostly_ capitalize
```suggestion
name: Mark Issues for Triage
```